README.md

```diff
@@ -90,11 +91,21 @@ Compares the version in package.json with the latest version on the registry.
 Usage: slt <CMD> [PKG]
 
 Commands:
-  lint      Perform a simple linting of the given package.json
-  cla       Create or verify contribution guidelines
-  info      Display metadata about package
-  version   Version manipulation
-  help      Print this usage guide
```

Comment:
the use of `[F]` is a bit hard for me to understand. I'm not really sure what I'm supposed to insert there `--flags` or something else?